.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+## Description
+(A few sentences describing the overall goals of the pull request's commits.)
```

Comment:
The hints in brackets may be surrounded by HMTL comment tags (i.e. `<!-- -->`) so that the contributor reads them when doing the PR but will prevent them from being rendered once the PR is opens. This has a few benefits:
- The contributor is not forced to delete them.
- The reviewers see only the relevant text.

e.g.
```
<!-- A few sentences describing the overall goals of the pull request's commits. -->
```

Reply:
Ah cool ! I didn't know that, thanks